packages/api/README.md

```diff
@@ -14,6 +14,75 @@ Includes the Kes configuration files needed for the deployment of the Api:
 - `config/api_v1.yml`: Includes all the Api endpoints and is versioned. Future versions of the Api should duplicate and rename this file
 - `config/distribution.yml`: Includes config needed for the distribution endpoint that handles file downloads with EarthLogin
 
+## Command Line Interface
+
+A command line interface is available to provide some additional API functionality. To see the available commands, install the Cumulus API package and run `cumulus-api --help`.
+
+### Reindexing Elasticsearch Indices
+
+Reindexing is available via the command line tool. Cumulus uses [index aliases](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html) to allow reindexing with no downtime. When using the reindex functionality, a new index will be created and all of the data will be indexed to the new index. When the operation is complete, the user can choose to switch to using the new index and delete the old index after verifying the new index.
+
+The Elasticsearch commands require a host to be specified. Your host can be found in AWS by going to the Elasticsearch Service and choosing your domain. The endpoint is the host. You can view your indices here as well on the indices tab.
+
+#### Reindex
+
+The reindex command creates a new index and reindexes the source index to the new, destination index. 
+
+An alias should not be specified unless you have a specific alias configured. If a source index is not specified, it will default to the index from the alias. If you want to name the destination index something particular, you can specify a name, otherwise the destination index name will default to 'cumulus-year-month-day' with today's date.
```

Comment:
what is an alias? I would add a short explanation and then also link to https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html

Reply:
I have that link on line 23. Do you think it needs more explanation or I need to include the link again?

Reply:
I was thinking it could be helpful to add a definition of an index alias, since it is, from my understanding, specific to elasticsearch. Something slightly less obtuse than the elasticsearch docs perhaps:

> The index aliases API allows aliasing an index with a name, with all APIs automatically converting the alias name to the actual index name

Perhaps:
> An index alias is a secondary name for an index and an index alias can map to more than one index. Read more at [Index Aliases](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html).